docs/extensions.md

```diff
@@ -222,7 +222,7 @@ Replace "**EXTENSION-NAME**" with the name of the extension.
 {
     "name": "EXTENSION-NAME",
     "type": "Microsoft.Resources/deployments",
-    "apiVersion": "[variables('apiVersionLinkDefault')]",
+    "apiVersion": "[variables('apiVersionDefault')]"

```

Comment:
why remove the "," here?

Reply:
whoops that was accidental, will fix thanks for catching